packages/web/src/components/NavBar/NavProfileMenu.tsx

```diff
@@ -10,28 +10,55 @@ import {
   Avatar,
 } from '@chakra-ui/react';
 
-export const NavProfileMenu: React.FC = () => (
-  <Menu variant="navbar">
-    <MenuButton as={Button}>
-      <Avatar size={'sm'} />
-    </MenuButton>
-    <MenuList>
-      <MenuItem>
-        <Link variant="navbarMenuItem" href="/app/profile">
-          Edit Profile
-        </Link>
-      </MenuItem>
-      <MenuItem>
-        <Link variant="navbarMenuItem" href="/app/contributions">
-          View Contributions
-        </Link>
-      </MenuItem>
-      <MenuDivider />
-      <MenuItem>
-        <Link variant="navbarMenuItem" href="/api/auth/github/logout">
-          Log Out
-        </Link>
-      </MenuItem>
+export const NavProfileMenu: React.FC = () => {
+  const [currentUserId, setCurrentUserId] = React.useState('');
+
+  React.useEffect(() => {
+    const fetchCurrentUserID = async () => {
+      const res = await fetch(`/api/users/me`);
+
+      if (res.status == 200) {
+        const data = await res.json();
+        setCurrentUserId(data.id);
+      }
+    };
+
+    fetchCurrentUserID();
+  }, []);
+
+  return (
+    <Menu variant="navbar">
+      <MenuButton as={Button}>
+        <Avatar size={'sm'} />
+      </MenuButton>
+<MenuList>
```

Comment:
Looks like the indention here got messed up; run `prettier:fix` and it should get cleaned up